README.md

```diff
@@ -25,3 +25,5 @@ export GITHUB_CREATE_REPO_ACCESS_TOKEN=xxxxx
 ```
 
 To be able to push to github without being prompted for username/password, set up git to use SSH
+`https://help.github.com/articles/connecting-to-github-with-ssh/`
```

Comment:
could you wrap it in a link like this?
`[set up git to use SSH](https://help.github.com/articles/connecting-to-github-with-ssh/)`